src/nexuscli/nexus_client.py

```diff
@@ -117,7 +117,10 @@ def rest_url(self):
 
         :rtype: str
         """
-        url = urljoin(self.config.url, '/service/rest/')
+        baseurl = self.config.url
```

Comment:
Wouldn't it be better to enforce this in the NexusConfig class?  i.e.: https://github.com/thiagofigueiro/nexus3-cli/blob/master/src/nexuscli/nexus_config.py#L45